### DISCUSSION TITLE
SGI STL contrib

### DISCUSSION THREAD

**Discussion by revelator** [open-watcom/open-watcom-v2#65] [Show and tell]
http://sourceforge.net/projects/cbadvanced/files/Open%20Watcom/sgi-stl-ow.7z/download

Gave up on stlport to many changes needed so instead i focussed on the old SGI STL,
it took some Work to iron out what was needed but i can report 100% success Building open watcom with it now so it should Work for anything else.

WxWidgets 2.10.12 also builds nicely with this.

**Comment by pchapin** [comment]
Thanks for your comment. I'm not clear what, if anything, you are asking here. Are you just letting us know about your port of SGI STL to Open Watcom? I wonder if there is a place on GitHub or SourceForge where links to such things can be displayed. Jiri?

**Comment by revelator** [comment]
Just to let people know, was not sure where else to post about it.

Though the SGI STL is rahther old by now it does implement some of the missing things from OW STL,
so if it helps getting those parts done its just another plus. In the meantime people can use it if they need that functionality.

**Comment by pchapin** [comment]
Okay, that's great. Thanks for the information!

**Comment by revelator** [comment]
All for a good cause :) i hope to see Open Watcom gain the same momentum MinGW and its 64 bit offshot has. Im still trying to draft more devs also though many are game devs some know a thing or two about compilers as well.
